packages/app/src/cli/constants.ts

```diff
@@ -60,7 +60,7 @@ export const publicFunctionExtensions = {
   types: ['product_discounts', 'order_discounts', 'shipping_discounts'],
```

Comment:
**Context for Guide Readers:** This is the only file that you need to change to prototype your API.